addons/mail/models/res_config_settings.py

```diff
@@ -42,6 +43,12 @@ class ResConfigSettings(models.TransientModel):
     )
     sfu_server_url = fields.Char("SFU Server URL", config_parameter="mail.sfu_server_url")
     sfu_server_key = fields.Char("SFU Server key", config_parameter="mail.sfu_server_key", help="Base64 encoded key")
+    sfu_local_key = fields.Char(
```

Comment:
Still have inconsistency here of having the field in the model but not in the view.
Either it should be in both or in none of them! (Probably none?)